binding/flutter/ios/Classes/SwiftRhinoPlugin.swift

```diff
@@ -118,40 +118,30 @@ public class SwiftRhinoPlugin: NSObject, FlutterPlugin {
     
     private func errorToFlutterError(_ error: RhinoError) -> FlutterError {
         switch(error) {
-        case .RhinoOutOfMemoryError:
-            return FlutterError(code: "RhinoMemoryException", message: extractMessage("\(error)"), details: nil)
-        case .RhinoIOError:
-            return FlutterError(code: "RhinoIOException", message: extractMessage("\(error)"), details: nil)
-        case .RhinoInvalidArgumentError:
-            return FlutterError(code: "RhinoInvalidArgumentException", message: extractMessage("\(error)"), details: nil)
-        case .RhinoStopIterationError:
-            return FlutterError(code: "RhinoStopIterationException", message: extractMessage("\(error)"), details: nil)
-        case .RhinoKeyError:
-            return FlutterError(code: "RhinoKeyException", message: extractMessage("\(error)"), details: nil)
-        case .RhinoInvalidStateError:
-            return FlutterError(code: "RhinoInvalidStateException", message: extractMessage("\(error)"), details: nil)
-        case .RhinoRuntimeError:
-            return FlutterError(code: "RhinoRuntimeException", message: extractMessage("\(error)"), details: nil)
-        case .RhinoActivationError:
-            return FlutterError(code: "RhinoActivationException", message: extractMessage("\(error)"), details: nil)
-        case .RhinoActivationLimitError:
-            return FlutterError(code: "RhinoActivationLimitException", message: extractMessage("\(error)"), details: nil)
-        case .RhinoActivationThrottledError:
-            return FlutterError(code: "RhinoActivationThrottledException", message: extractMessage("\(error)"), details: nil)
-        case .RhinoActivationRefusedError:
-            return FlutterError(code: "RhinoActivationRefusedException", message: extractMessage("\(error)"), details: nil)
-        case .RhinoInternalError:
-            return FlutterError(code: "RhinoException", message: extractMessage("\(error)"), details: nil)
+        case .RhinoOutOfMemoryError (let message):
```

Comment:
Is there any way to just case the RhinoError type to a string? It looks like if we could do that, we could remove this function altogether.